packages/tldraw/src/lib/ui/components/Toasts.tsx

```diff
@@ -80,36 +82,16 @@ function Toast({ toast }: { toast: TLUiToast }) {
 	)
 }
 
-function _Toasts() {
-	const { toasts } = useToasts()
-
-	return toasts.map((toast) => <Toast key={toast.id} toast={toast} />)
-}
-
-export const Toasts = React.memo(_Toasts)
-
-export function ToastViewport() {
-	const editor = useEditor()
+/** @public @react */
+export const TldrawUiToasts = memo(function TldrawUiToasts() {
```

Comment:
Should unify the naming here. We use `TldrawUiToasts`, but then we use just `Toast` above. Might also make sense to update the filename to reflect the change?

Reply:
We could, the naming only really matters for exported things (we don't export Toast) but as a convention we might as well